app/views/software_choices.scala.html

```diff
@@ -0,0 +1,96 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@(softwareChoices: SoftwareChoicesViewModel)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = messages("softwareChoices.title"), bodyClasses = None) {
+    <h1>@messages("softwareChoices.title")</h1>
+
+    <div class="form-group">
```

Comment:
Refactor this search form out into it's own template so that the search is self contained and can be reused if needed